org.freeplane.App.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.freeplane.App
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+  - --allow=multiarch
+
+  - --env=PATH=/usr/bin:/app/bin:/app/jdk/bin
+  - --env=JAVA_HOME=/app/jre
+  - --env=JAVA=/app/jre/bin/java
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk11/
+        freeplane_VERSION: '1.8.1'
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: freeplane
+    no-autogen: true
+    sources:
+      # Fetch project.
+      - type: archive
+        url: https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_bin-1.8.1.zip
+        sha256: faae9cd9fd5a7fc19057737d1c3cf3d107c11b8b8d9f9b59069c95d9d7d5f673
+      
+      - type: file
+        path: org.freeplane.App.Makefile
+        dest-filename: Makefile
+
+      - type: file
+        path: org.freeplane.App.png
+      
+        # We need to change the  WORKINGDIRECTORY,
+        # otherwise it will attempt to write in the installation dir.
+      - type: patch
+        path: freeplane-launcher.patch
+      
+      # Install AppData.
+      - type: file
+        path: org.freeplane.App.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html